packages/components/toggle-button/src/toggle-button.stories.ts

```diff
@@ -85,93 +105,98 @@ export const All: Story = {
   render: () => {
     return html`
       <style>
-        sl-button-bar {
-          margin-block-end: 8px;
-        }
-        table {
-          border-collapse: collapse;
+        #root-inner {
+          align-items: center;
+          align-self: start;
+          display: inline-grid;
+          gap: 1rem 2rem;
+          grid-template-columns: auto 1fr 1fr 1fr 1fr;
+          justify-items: center;
         }
+      </style>
+      <span>Ghost</span>
```

Comment:
why not use a table for this?

Reply:
Grid is less markup than `<table>`. Also allows for more flexibility (see the "All" story of toggle-group)

Reply:
But in this case, perhaps the difference is less clear